posts/templates/posts/post_detail.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+{% load static %}
+{% block styles %}
+    <link rel="stylesheet" type="text/css" href="{% static 'posts/feed.css' %}">
+{% endblock styles %}
+{% block content %}
+    <article class="content-section">
+        <div class="media-body">
+        <div class="article-metadata">
+            <img class="rounded-circle article-img" src="{{ post.author.profile.profile_pic.url }}">
+            <a class="mr-2" href="#">{{ post.author }}</a>
+            <small class="text-muted">{{ post.date_posted|date:"F d, Y" }}</small>
+        </div>
+        <p class="article-content">{{ post.description }}</p>
+            {% if post.resume %}
+            <iframe src="{{ post.resume.resume_file.url }}" style="width: 560px;height: 780px ;border: none;"></iframe>
+            <hr/>
```

Comment:
Please replace this line as @natifridman suggested to @AlonShakaroff in his PR
```suggestion
            <hr>
```
In my opinion, you can even remove it completely.
I don't think a separator line is necessary for the detail view (because only one post is being displayed).

Reply:
Done